src/shared/support-prompt.ts

```diff
@@ -141,6 +141,11 @@ Please provide:
 2. Explanation of each part/flag
 3. Expected output and behavior`,
 	},
+	NEW_TASK: {
+		label: "Start New Task",
+		description: "Start new task with user input",
+		template: `\${userInput}`,
+	},
```

Comment:
Does it need to be added here?

Reply:
Just thinking that this adds another tab at the bottom of the prompts view right?

Reply:
In my scenario, other plugins will send tasks to Roo, and I hope Roo can define its own prompt words when receiving instructions from other plugins.

In addition, because this pr also supports users to create tasks directly through the command palette, this option can also be used for users to define their own "quick start" prompt words.

Reply:
Does this work for the problem you're trying to solve?
https://github.com/RooVetGit/Roo-Code/blob/main/src/exports/roo-code.d.ts#L31

This should allow other extensions to interface with Roo Code. Or is it more convenient to use an action?

Reply:
I think command is more convenience. it can be called from the command palette, shortcut keys, other plugins or the command line